### ISSUE TITLE
Feature Request: Use AzGovViz without ManagementGroup.Read permission

### ISSUE THREAD

**Issue by mschwrdtnr** [JulianHayward/Azure-MG-Sub-Governance-Reporting#214]
### Problem / Challenge
Because of corporate regulations, it's not always possible to get permissions on the necessary ManagementGroup. 

### Desired solution
I would like to use the tool 
- to scan just with Subscription Reader permissions
- to just scan based on a subscription ID

## Questions
1. Is this possible?
2. Is there another way to use the scanner with just Reader permissions on Subscription level or a way to implement this? What are resulting restrictions?

**Comment by JulianHayward** [comment]
@mschwrdtnr there is currently no plan to strip down the tool to only process subscriptions. Have you already checked the [Azure Quick Review](https://github.com/Azure/azqr#azure-quick-review) (azqr) tool?

**Comment by mschwrdtnr** [comment]
@JulianHayward alright, thanks for the answer! If you have things in your mind how this can be implemented and you think this is a meaningful feature, I would be happy about some suggestions to check if I am able to implement.

Yes, I already checked and tested azqr and also [PSRule.Azure](https://github.com/Azure/PSRule.Rules.Azure/). Azqr does have much less checks than PSRule.Azure have. It also has not that many capabilities like your tool. The only advantage is the good excel export, but I am currently working on getting better outputs of the PSRule checker. 

I already started a [discussion in azqr](https://github.com/Azure/azqr/discussions/148) about the differences to other tools. I see many possible review tools where people, in my view, aren't really work together 🤔  Azqr with the checks of PSRule would be insane.
